.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Invalid JSON? Does this matter or does VScode handle this

Reply:
Yeah, technically it is json5 but .vscode/launch.json comes with these comments so it's "correct" for this purpose